metricflow-semantics/metricflow_semantics/mf_logging/pretty_print.py

```diff
@@ -60,7 +60,10 @@ def _handle_sequence_obj(
         elif isinstance(list_like_obj, set) or isinstance(list_like_obj, frozenset):
             left_enclose_str = f"{type(list_like_obj).__name__}("
             right_enclose_str = ")"
-            list_like_obj = sorted(list_like_obj)
+            list_like_obj = sorted(
+                list_like_obj,
+                key=(lambda x: x.name) if (list_like_obj and isinstance(list(list_like_obj)[0], Enum)) else None,
+            )
```

Comment:
This will be more robust - albeit slower - if we do this instead:

```suggestion
            list_like_obj = sorted(self._handle_any_obj(obj) for obj in list_like_obj)
```